app/models/optout/newJourney/OptOutTaxYearQuestionViewModel.scala

```diff
@@ -35,7 +35,13 @@ case class OptOutTaxYearQuestionViewModel(taxYear: OptOutTaxYear,
     case _: NextOptOutTaxYear     => "nextYear"
   }
 
-  private val optOutStateMessageSuffix = (optOutState, hasNoQuarterlyUpdatesSubmitted, currentYearStatus, nextYearStatus) match {
+  private val effectiveOptOutState: Option[OptOutState] = optOutState match {
+    case Some(MultiYearOptOutDefault) if isPreviousYear && (currentYearStatus == Mandated | currentYearStatus == ITSAStatus.Dormant) => Some(OneYearOptOutFollowedByMandated)
+    case Some(MultiYearOptOutDefault) if isCurrentYear && (nextYearStatus == Mandated | nextYearStatus == ITSAStatus.Dormant) => Some(OneYearOptOutFollowedByMandated)
+    case _ => optOutState
+  }
+
+  private val optOutStateMessageSuffix = (effectiveOptOutState, hasNoQuarterlyUpdatesSubmitted, currentYearStatus, nextYearStatus) match {
```

Comment:
just a suggestion but wondering if structuring the logic like this would make it more concise?

```
  private val effectiveOptOutState: Option[OptOutState] =
    (optOutState, currentYearStatus, nextYearStatus) match {
      case (Some(MultiYearOptOutDefault), Mandated, Dormant) if isPreviousYear =>
        Some(OneYearOptOutFollowedByMandated)
      case (Some(MultiYearOptOutDefault), Mandated, Dormant) if isCurrentYear =>
        Some(OneYearOptOutFollowedByMandated)
      case _ =>
        optOutState
    }
```

moving the logic away from the guard.

Reply:
🤦‍♂️ sorry looking back yeah you're right, I kinda screwed it up, would this work?

```

  private val effectiveOptOutState: Option[OptOutState] =
    (optOutState, currentYearStatus, nextYearStatus) match {
      case (Some(MultiYearOptOutDefault), Mandated | Dormant, _) if isPreviousYear =>
        Some(OneYearOptOutFollowedByMandated)
      case (Some(MultiYearOptOutDefault), _, Mandated | Dormant) if isCurrentYear =>
        Some(OneYearOptOutFollowedByMandated)
      case _ =>
        optOutState
    }
  
```